jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/editor/LogGroupTable.kt

```diff
@@ -53,9 +54,15 @@ class LogGroupTable(
     }
 
     init {
+        val (sortColumn, sortOrder) = when (type) {
+            TableType.LIST -> 1 to SortOrder.DESCENDING
+            TableType.FILTER -> 1 to SortOrder.ASCENDING
```

Comment:
Changed to ASCENDING since name should be alphabetical sorting and thus it makes more sense to be an alphabetical order and not reversed

Reply:
Ah. true